reccmp/isledecomp/compare/db.py

```diff
@@ -441,6 +457,15 @@ def recomp_used(self, addr: int) -> bool:
         cur = self._sql.execute("SELECT 1 FROM entities WHERE recomp_addr = ?", (addr,))
         return cur.fetchone() is not None
 
+    def used(self, img: ImageId, addr: int) -> bool:
+        if img == ImageId.ORIG:
+            return self.orig_used(addr)
+
+        if img == ImageId.RECOMP:
+            return self.recomp_used(addr)
+
+        return False
```

Comment:
If it's necessary to write anything here (it should be unreachable from a type perspective, but maybe it's still required), I'd `assert False` here